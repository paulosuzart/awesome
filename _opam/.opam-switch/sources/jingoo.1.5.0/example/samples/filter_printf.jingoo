{{ printf ("%d %s, %d people, that's %.1f apple per person.", 8, "apples", 4, 0.5) }}
