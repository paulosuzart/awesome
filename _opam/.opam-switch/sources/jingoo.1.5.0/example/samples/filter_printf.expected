8 apples, 4 people, that&apos;s 0.5 apple per person.
